addons/binding/org.openhab.binding.te923/README.md

```diff
@@ -0,0 +1,49 @@
+# TE923, Irox Binding
+
+This is an openHAB binding for most weather stations based on Hideki weather station like IROX Pro X, Mebus TE923 or TFA Nexus. Some other hardware mays be supported.
```

Comment:
TE923 seems a specific model, so it's more logical to rename the binding to something like: `hidekiweatherstation` Even while the exec used here uses the name te923 that name is to specific a single variant, while it seem to support other models too.. But @martinvw or @kaikreuzer  can better give provide a guide to the naming.

Reply:
TE923 seams to be the "white label" of Hideki, then rebranded by some other company.
http://www.hidekielectronics.com/?m=6&p=2